scripts/launch/node.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+# directory with script
+base=$(dirname "$0")
+# project root
+root="$base/../.."
+
+# permanent read-only files: configs, secrets...
+files="$root/run"
+# gitignored files: databases...
+tmp_files="$files/tmp/"
+
+mkdir $tmp_files 2> /dev/null || true
+
+##################
+# Parsing params
+##################
+
+for var in "$@"
+do
+    if [[ "$var" == "witness" ]] || [ "$var" == "w" ]; then
+        mode="witness"
+    elif [[ "$var" == "witness" ]] || [ "$var" == "e" ]; then
+        mode="educator"
+    else
+        echo "Unknown parameter \"$var\", ignoring"
+    fi
+done
+
+##################
+# Launch
+##################
+
+# educator-only params
+educator_params="
+--keyfile-password 12345678
+--keyfile-path $files/educator.key
+--sql-path $tmp_files/educator.db
+"
+
+# witness params (and educator's as well)
+witness_params="
+--bind 127.0.0.1:4010:4011
+--db-path $tmp_files/witness.db
+--log-dir $tmp_files/logs
+"
+
+if [[ "$mode" == "educator" ]]; then
+    stack exec "disciplina-educator" -- $witness_params $educator_params
+elif [[ "$mode" == "witness" ]]; then
+    stack exec "disciplina-witness" -- $witness_params
+elif [[ "$mode" == "" ]]; then
+    echo "Please specify which node to run (e.g. \"educator\" or just \"e\")"
```

Comment:
`'s/node/mode/'`

Reply:
That is so for purpose, I see it like we have educator node and witness node.

Reply:
Right, set it `node` everywhere